.github/workflows/charts.yml

```
@@ -78,7 +78,7 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@b5ca514318bd6ebac0fb2aedd5d36ec1b5c232a2 # v3.10.0
         with:
-          version: v0.16.2
+          version: latest
```

```suggestion
          version: v0.22.0
```
per https://github.com/docker/buildx/releases/tag/v0.22.0

---

updated.